### ISSUE TITLE
Dropdown: Narrator is not announcing the options when using Up/Down arrow keys (without the list being expanded)

### ISSUE THREAD

**Issue by iwasn** [OfficeDev/office-ui-fabric-react#10497]
### Environment Information

- **Package version(s)**: Latest (7.35.0)
- **Browser and OS versions**: Windows/Edge (but possibly all)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Please provide a reproduction of the issue in a codepen:

* Go to dropdown page using narrator: https://fabricweb.z5.web.core.windows.net/oufr/7.35.0/#/examples/dropdown

* Focus on any dropdown via tab

* Expand dropdown (via alt-down, then selecting any element)

* Still with focus on the dropdown, while not expanded, use Up/Down keys to move through the items in place.

#### Actual behavior:

Narrator should announce updated item in the dropdown.

#### Expected behavior:

Narrator is not announcing the updated item in the dropdown.

**Comment by aneeshack4** [comment]
@iwasn Apologies for the delay - my surface book was acting up but it's fixed now so I was finally able to repro this. This repros in scan mode and when scan mode's off too. It's a dupe of this [Dropdown issue](https://github.com/OfficeDev/office-ui-fabric-react/issues/2654) which seems to have been fixed but maybe this is a regression. Will speak with the component owner/person who fixed the issues originally & report back.

**Comment by aneeshack4** [comment]
Looks like this is actually a regression - #7357 back in Dec 18 2018 regressed this. Will send out a fix soon.

**Comment by natalieethell** [comment]
Thanks for reporting this @iwasn! The main change in the PR that Aneesha linked above was only setting `aria-activedescendant` when the dropdown was expanded. So, when the dropdown is collapsed, the focus now goes on the outermost dropdown container instead of the option itself. I'm still not sure what the right fix would be to retain both behaviors, but that's what I think the issue is.

**Comment by iwasn** [comment]
@aneeshack4 Thanks for the fix! Since this is closed, is the fix already in the latest version that we can merge in?

**Comment by aneeshack4** [comment]
@iwasn it just got merged so it'll be in the version that gets published tonight!
